tensorflow_model_optimization/g3doc/guide/clustering/clustering_example.ipynb

```diff
@@ -0,0 +1,515 @@
+{
```

Comment:
<p>1. For submission, please stay with the latest stable TF release (! pip install tensorflow).</p><p> </p><p>2. Once you and Aron find things code complete, you're welcome to create a PR to expose clustering in the public API (which we won't merge until closer to launch) and I can create a test TFMOT pip package for you to use. I'll also do one more thorough pass through the tests and code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Thanks,</p><blockquote>&nbsp;to create a PR to expose clustering in the public API&nbsp;</blockquote><p>Not sure what that means, is there any example of this?</p><p> </p>

Reply:
Expose clustering in https://github.com/tensorflow/model-optimization/tree/master/tensorflow_model_optimization/python/core/api like the quantization example (don't follow pruning - not best practices yet). 

Afterwords, if you build the pip from source, you should be able to import clustering like:

import tensorflow_model_optimization as tfmot
tfmot.clustering.keras.cluster_weights()

Reply:
Resolving since Aron created a PR for exposing clustering in the public API.